File: patterns/C++/pattern5.cpp
/*
Generate the following pattern
* * * * *
* * * *
* * *
* *
*
*/

#include<bits/stdc++.h>
using namespace std;

void pattern5(int n){
    for(int i=1; i<=n; i++){
        for(int j=n; j>=i; j--){
            cout << "* ";
        }
        cout << "\n";
    }
}

int main() {
    int n;
    cout << "Enter the number of rows(n): ";
    cin >> n;
    cout << "Here is the generated pattern\n";
    pattern5(n);

}

File: patterns/Python/pattern5.py
"""
Generate the following pattern
* * * * *
* * * *
* * *
* *
*
"""

def pattern5(n):
    for i in range(n):
        for j in range(n,i,-1):
            print("* ", end="")
        print("")
        

def main():
    i = int(input("Enter the number of rows: "))
    print("Here is the generated pattern\n")
    pattern5(i)
    
if __name__ == "__main__":
    main()